lib/pbench/server/api/resources/upload_api.py

```diff
@@ -79,26 +79,33 @@ def put(self, controller):
         md5sum = request.headers.get("Content-MD5")
 
         if not self.allowed_file(filename):
-            self.logger.debug(
+            self.logger.warning(
                 f"Tarfile upload: Bad file extension received for file {filename}"
             )
             abort(400, message="File extension not supported. Only .xz")
 
+        content_length = request.headers.get("Content-Length")
+        if not content_length:
+            self.logger.warning(
+                f"Tarfile upload: No Content-Length header value found for file {filename}"
+            )
+            abort(400, message="Missing required content-length header")
```

Comment:
Or we could use "411 Length Required"

Reply:
Or, eh-hem, we could use `HTTPStatus.LENGTH_REQUIRED`...